### PR TITLE
feat: add pay-as-you-go plan card

### PR DESCRIPTION
**Pull request by bryanoltman**
## Status

**IN DEVELOPMENT**

## Description

Updates the teams card for the pay-as-you-go plan

Screenshot 2024-08-26 at 4 31 58 PM

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by bryanoltman** [comment]
The copy obviously needs work, but here's a first draft 😄

**Comment by felangel** [comment]
I feel like it might be good to leave it as the teams plan and just adjust the details to include pay as you go. It’s still a teams plan in the sense that it’s designed for teams that want to collaborate right?

**Comment by bryanoltman** [comment]
@felangel Yeah, that would make sense. I think we generally (here and in the announcement) should align on the positioning of the new plan (e.g., will we present it to customers as a new version of the Teams plan?)

**Comment by felangel** [comment]
> @felangel Yeah, that would make sense. I think we generally (here and in the announcement) should align on the positioning of the new plan (e.g., will we present it to customers as a new version of the Teams plan?)

Yeah agreed. I think it makes sense for this company I be a new and improved teams plan 🤷‍♂️

File: src/components/Pricing.tsx [hunk 1]
```diff
@@ -136,6 +137,52 @@ function PriceSlider({
   );
 }
 
+function PayAsYouGoPlan() {
```
Comment (felangel) [inline src/components/Pricing.tsx:140]:
```suggestion
function TeamPlanV2() {
```

For consistency 🤷

File: src/components/Pricing.tsx [hunk 2]
```diff
@@ -136,6 +137,52 @@ function PriceSlider({
...
+              <FeatureListItem
+                key={`hobby-feature-${index}`}
+                title={feature.title}
+              />
+            ))}
+          </ul>
+          <a
+            target="_blank"
+            href={config.consoleUrl}
+            className="plausible-event-name=Pay+as+you+go+Get+Started+Button+Clicked shorebird-button-primary mt-8 inline-block w-full rounded-xl rounded-t-xl px-4 py-2 text-center font-bold leading-loose"
```
Comment (felangel) [inline src/components/Pricing.tsx:176]:
```suggestion
            className="plausible-event-name=teams+v2+Get+Started+Button+Clicked shorebird-button-primary mt-8 inline-block w-full rounded-xl rounded-t-xl px-4 py-2 text-center font-bold leading-loose"
```

File: src/components/Pricing.tsx (hunk 1 above)
Comment (bryanoltman) [reply src/components/Pricing.tsx]: Done

File: src/components/Pricing.tsx (hunk 2 above)
Comment (bryanoltman) [reply src/components/Pricing.tsx]: Done

File: src/components/Pricing.tsx [hunk 3]
```diff
@@ -136,6 +137,52 @@ function PriceSlider({
   );
 }
 
+function TeamPlanV2() {
+  const features: Feature[] = [
+    { title: 'Unlimited apps' },
+    { title: 'Unlimited developers' },
+    { title: '50K patch installs/month included' },
+    { title: 'Control exactly how much you spend' },
```
Comment (felangel) [inline src/components/Pricing.tsx:145]:
```suggestion
    { title: 'Configurable spending limit' },
```

🤷

**Review by felangel — APPROVED** [review]

File: src/components/Pricing.tsx [hunk 4]
```diff
@@ -136,6 +137,52 @@ function PriceSlider({
...
+          <p className="mb-4 text-left text-xs leading-loose text-gray-500">
+            then $1 per 2,500 patch installs.
+          </p>
+          <p className="mb-4 mt-4 text-left leading-loose text-gray-500">
+            For apps that can scale.
+          </p>
+          <ul className="mb-2 text-white 2xl:mb-6">
+            {features.map((feature, index) => (
+              <FeatureListItem
+                key={`hobby-feature-${index}`}
```
Comment (felangel) [inline src/components/Pricing.tsx:168]:
```suggestion
                key={`team-v2-feature-${index}`}
```

File: src/components/Pricing.tsx (hunk 3 above)
Comment (bryanoltman) [reply src/components/Pricing.tsx:145]: @eseidel thoughts on this?

Comment (eseidel) [reply src/components/Pricing.tsx:145]: I'm not sure we need to update the copy for teams at all?  AT least in terms of the benefits.  "PAYG" isn't really a feature so much as "table stakes" to those coming to us from never having heard of us.

File: src/components/Pricing.tsx [hunk 5]
```diff
@@ -136,6 +138,52 @@ function PriceSlider({
...
+          >
+            Get Started
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
 function TeamPlan() {
```
Comment (eseidel) [inline src/components/Pricing.tsx:187]: dead code.

File: src/components/Pricing.tsx [hunk 6]
```diff
@@ -36,7 +37,8 @@ export const Pricing = () => {
             </div>
             <div className="mt-20 flex flex-col flex-wrap items-center justify-center gap-4 lg:flex-row">
               <HobbyPlan />
-              <TeamPlan />
+              {/* <TeamPlan /> */}
```
Comment (felangel) [inline src/components/Pricing.tsx:40]: Do we want to just delete the team plan?

**Review by eseidel — APPROVED** [review]

File: src/components/Pricing.tsx (hunk 6 above)
Comment (bryanoltman) [reply src/components/Pricing.tsx]: Deleted

File: src/components/Pricing.tsx (hunk 5 above)
Comment (bryanoltman) [reply src/components/Pricing.tsx]: Yep, deleted

**Pull request merged**